src/OrchardCore/OrchardCore.Data.YesSql/Removing/ShellDbTablesInfo.cs

```diff
@@ -0,0 +1,298 @@
+using System;
+using System.Collections.Generic;
+using System.Data.Common;
+using System.Linq;
+using Dapper;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+using YesSql;
+using YesSql.Services;
+using YesSql.Sql;
+using YesSql.Sql.Schema;
+
+namespace OrchardCore.Environment.Shell.Removing;
+
+internal class ShellDbTablesInfo : ISchemaBuilder
+{
+    private ICommandInterpreter _commandInterpreter;
+    public string DatabaseProvider { get; set; }
```

Comment:
Shouldn’t this use [DatabaseProviderName ] (https://github.com/OrchardCMS/OrchardCore/blob/b4188902be9c77d09455494f4fce1c58ec44d39d/src/OrchardCore/OrchardCore.Data.Abstractions/DatabaseProviderName.cs) enum instead of a string?

Reply:
Not used internally, just to hold the value of `ShellSettings["DatabaseProvider"]` which is a string. Anyway I removed it, was just to be able externally to use `tablesInfo.DatabaseProvider` in place of  `ShellSettings["DatabaseProvider"]` which I now use when comparing to a given provider. I also created a new static class for provider names const as strings to not have to deal with an enum.

Reply:
Yes I saw internal class. just thought its best to use `DatabaseProviderName` for consistency. Thanks for changing that